aws/resource_aws_elasticache_replication_group_test.go

```diff
@@ -1115,122 +1213,116 @@ resource "aws_elasticache_replication_group" "bar" {
 
 var testAccAWSElasticacheReplicationGroupRedisClusterInVPCConfig = fmt.Sprintf(`
 data "aws_availability_zones" "available" {
-  # InvalidParameterValue: Specified node type cache.m3.medium is not available in AZ us-east-1b.
-  blacklisted_zone_ids = ["use1-az1"]
-  state                = "available"
-}
-
-resource "aws_vpc" "foo" {
-    cidr_block = "192.168.0.0/16"
-  tags = {
-        Name = "terraform-testacc-elasticache-replication-group-redis-cluster-in-vpc"
-    }
-}
-
-resource "aws_subnet" "foo" {
-    vpc_id = "${aws_vpc.foo.id}"
-    cidr_block = "192.168.0.0/20"
-    availability_zone = "${data.aws_availability_zones.available.names[0]}"
-  tags = {
-        Name = "tf-acc-elasticache-replication-group-redis-cluster-in-vpc-foo"
-    }
-}
-
-resource "aws_subnet" "bar" {
-    vpc_id = "${aws_vpc.foo.id}"
-    cidr_block = "192.168.16.0/20"
-    availability_zone = "${data.aws_availability_zones.available.names[1]}"
-  tags = {
-        Name = "tf-acc-elasticache-replication-group-redis-cluster-in-vpc-bar"
-    }
-}
-
-resource "aws_elasticache_subnet_group" "bar" {
-    name = "tf-test-cache-subnet-%03d"
-    description = "tf-test-cache-subnet-group-descr"
-    subnet_ids = [
-        "${aws_subnet.foo.id}",
-        "${aws_subnet.bar.id}"
-    ]
-}
-
-resource "aws_security_group" "bar" {
-    name = "tf-test-security-group-%03d"
-    description = "tf-test-security-group-descr"
-    vpc_id = "${aws_vpc.foo.id}"
-    ingress {
-        from_port = -1
-        to_port = -1
-        protocol = "icmp"
-        cidr_blocks = ["0.0.0.0/0"]
-    }
-}
-
-resource "aws_elasticache_replication_group" "bar" {
-    replication_group_id = "tf-%s"
-    replication_group_description = "test description"
-    node_type = "cache.m3.medium"
-    number_cache_clusters = "2"
-    port = 6379
-    subnet_group_name = "${aws_elasticache_subnet_group.bar.name}"
-    security_group_ids = ["${aws_security_group.bar.id}"]
-    availability_zones = ["${data.aws_availability_zones.available.names[0]}","${data.aws_availability_zones.available.names[1]}"]
-    automatic_failover_enabled = false
-    snapshot_window = "02:00-03:00"
-    snapshot_retention_limit = 7
-    engine_version = "3.2.4"
-    maintenance_window = "thu:03:00-thu:04:00"
-}
-`, acctest.RandInt(), acctest.RandInt(), acctest.RandString(10))
+	# InvalidParameterValue: Specified node type cache.m3.medium is not available in AZ us-east-1b.
```

Comment:
Nit: Looks like the formatting of this configuration got switched from 2 spaces to tabs